requirements.txt

```diff
@@ -1,4 +1,4 @@
-pygame>=2.1.2
+pygame-ce>=2.1.3
 click>=8.1.3
 colorama>=0.4.4
-keyboard>=0.13.5
\ No newline at end of file
+keyboard>=0.13.5
```

Comment:
No clue what's going on with this last line, probably something about a newline getting removed or maybe added.

Reply:
You can enable an option to hide this kind of changes on Github preview:
![image](https://user-images.githubusercontent.com/25286578/218721592-73b02c67-db67-4c18-8c93-d417325b0a10.png)
Then you can focus on the meaningful changes.

Reply:
Thanks for the tip. 👍